bfvmm/src/hve/arch/intel_x64/xen/xen_op.cpp

```diff
@@ -913,6 +1132,7 @@ xen_op_handler::SCHEDOP_yield_handler(gsl::not_null<vcpu *> vcpu)
     try {
         m_sched_op->handle_yield(vcpu);
         vcpu->set_rax(SUCCESS);
+        m_vcpu->return_and_continue();
```

Comment:
A yield would not be a return and continue. A yield should be a return and wait

Reply:
Yeah that stuff is old now; it will likely all be changed once we see sched_ops in there proper place